.golangci.yml

```
@@ -32,6 +32,19 @@ linters-settings:
     lines: 160
     statements: 70
 
+  # https://github.com/daixiang0/gci
+  # ensure import order is consistent
+  # gci write --custom-order -s standard -s default -s blank -s dot -s alias -s "prefix(github.com/go-vela)" .
+  gci:
+    custom-order: true
+    sections:
+      - standard
+      - default
+      - blank
+      - dot
+      - alias
+      - prefix(github.com/go-vela)
```

```suggestion
  # gci write --custom-order -s standard -s default -s blank -s dot -s "prefix(github.com/go-vela)" .
  gci:
    custom-order: true
    sections:
      - standard
      - default
      - blank
      - dot
      - prefix(github.com/go-vela)
```
Speaking of opinions: I don't like separating aliased imports from the other groups. What about skipping that section?

---

I'm good with that!

---

diff here https://github.com/go-vela/server/pull/1077/commits/dd8726309b37301b94c8d2f39c497e9278a1a734 with alias taken out